include/zephyr/arch/posix/linker.ld

```diff
@@ -83,6 +82,11 @@ SECTION_DATA_PROLOGUE(_NOINIT_SECTION_NAME,,)
 
  } INSERT AFTER .data;
 
+SECTIONS
+ {
+#include <zephyr/arch/posix/native_sim_interface.ld>
+ } INSERT AFTER .text;
```

Comment:
stray leading space?
```suggestion
{
#include <zephyr/arch/posix/native_sim_interface.ld>
} INSERT AFTER .text;
```

Reply:
oh, I see now the previous one has it as well... that is so weird it's like a backward indentation, should that be fixed? it looks off... fine if you think it's ok

Reply:
Changed. AFAIR that just got inherited over time and over files.

Reply:
thanks